features/lwipstack/mbed_lib.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "lwip",
     "config": {
+        "present": 1,
```

Comment:
What does this change exactly?

Reply:
It creates `#define MBED_CONF_LWIP_PRESENT 1`.
If a library includes the `lwIP` library it will define that macro and thus will compile in the code that is gated with `#if MBED_CONF_LWIP_PRESENT` so it can be excluded for baremetal builds.

In summary, without this **non-baremetal** builds will always fail if the targets rely on lwIP.